### PR TITLE
lyricsources: added subtitles4songs.

### PR DESCRIPTION
**Pull request by dasj19**
I added this new source to the collection, I think it is relatively easy to extract lrc from their website.

I would like some feedback as this is one of my first contributions to this project and my first time writing anything serious in python.

TODO:
* Remove the print statements before merge
* Squash all commits to the branch into a single one

### CODE REVIEW

File: lyricsources/subtitles4songs/subtitles4songs.py [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+        # type: (Any) -> bytes
+        status, content = http_download(downloadinfo,
+                                        proxy=get_proxy_settings(self.config_proxy))
+        if status < 200 or status >= 400:
+            raise http.client.HTTPException(status)
+        if content:
+            print('content:'); print(content)
+            content = S4S_LRC_PATTERN.search(content.decode('utf8')).group(2)
+            # Replacing html breaks with new lines.
+            content = re.sub('<br />', "\n", content)
```
Comment (PedroHLC) [inline lyricsources/subtitles4songs/subtitles4songs.py:102]: :thinking:  I didn't found any of those in the results I've inspected from `getsubtitle.aspx`, do you have an example?

**Review by PedroHLC — COMMENTED** [review]
:sweat_smile:  I forgot how this project prints instead of properly logging and it probably needs a "Copyright refactoring" as Tiger gave us the rights when created this organization.

I'll try building this in my lunchtime, so far the code looks logically right...

File: lyricsources/subtitles4songs/subtitles4songs.py (hunk 1 above)
Comment (dasj19) [reply lyricsources/subtitles4songs/subtitles4songs.py:102]: I think every single one of them has `<br />` tags.

Try `view-source:https://www.rentanadviser.com/en/subtitles/getsubtitle.aspx?artist=David%20Guetta&song=Titanium%20ft.%20Sia` (remember to look at the source code and not the rendered page)

Comment (PedroHLC) [reply lyricsources/subtitles4songs/subtitles4songs.py:102]: Aaaaaaa, you're crawling the website, while I was trying downloading it with a session:

```
curl 'https://www.rentanadviser.com/en/subtitles/getsubtitle.aspx?artist=David+Guetta&song=Titanium+ft.+Sia&type=lrc' -H 'User-Agent: Mozilla/5.0 (X11; Linux x86_64; rv:90.0) Gecko/20100101 Firefox/90.0' -H 'Accept: text/html,application/xhtml+xml,application/xml;q=0.9,image/webp,*/*;q=0.8' -H 'Accept-Language: en-US,en;q=0.8,pt-BR;q=0.5,es;q=0.3' --compressed -H 'Content-Type: application/x-www-form-urlencoded' -H 'Origin: https://www.rentanadviser.com' -H 'Connection: keep-alive' -H 'Referer: https://www.rentanadviser.com/en/subtitles/getsubtitle.aspx?artist=David%20Guetta&song=Titanium%20ft.%20Sia&type=lrc' -H 'Cookie: ASP.NET_SessionId=v1x3q5rp3hpu4g1jr25g3uvk' -H 'Upgrade-Insecure-Requests: 1' -H 'Sec-Fetch-Dest: document' -H 'Sec-Fetch-Mode: navigate' -H 'Sec-Fetch-Site: same-origin' -H 'Sec-Fetch-User: ?1' -H 'Pragma: no-cache' -H 'Cache-Control: no-cache' -H 'TE: Trailers' --data-raw '__EVENTTARGET=ctl00%24ContentPlaceHolder1%24btnlyrics&__EVENTARGUMENT=&ctl00%24ContentPlaceHolder1%24txtsearchsubtitle=Search+Lyrics+%26+Subtitles&ctl00%24ContentPlaceHolder1%24Overcome_Enter_problem_in_IE2=&__VIEWSTATEGENERATOR=2245B277&__EVENTVALIDATION=%2FwEdAAaHOmx%2Bh7BWQCNEhvgO%2BUE2RuJmd8xHp1FvBUnaqp%2B06DOW6IPr5umq7%2FycngDpTXM1RCHQqe6x74z2AXzs4B32JjCeau28td%2F8t%2FqvcxlGDE8RmFoVYBUtr01iOVUfHho2tBGalsBI5fy71AiczchqRqMTxNmjo8fByi56acZTOg%3D%3D&__VIEWSTATE=%2FwEPDwUKLTUxMDQ1NjY2NQ8WBh4JUm9vdGhQYXRoBUNDOlxJbmV0cHViXHZob3N0c1xyZW50YW5hZHZpc2VyLmNvbVxodHRwZG9jc1xlblxBcHBfRGF0YVxzdWJzXGRhdGFcHgdzdWJQYXRoBWRDOlxJbmV0cHViXHZob3N0c1xyZW50YW5hZHZpc2VyLmNvbVxodHRwZG9jc1xlblxBcHBfRGF0YVxzdWJzXGRhdGFcRGF2aWQtR3VldHRhLVRpdGFuaXVtLWZ0Li1TaWEuc3J0HglQYWdlUGFybXMFOT9hcnRpc3Q9RGF2aWQlMjBHdWV0dGEmc29uZz1UaXRhbml1bSUyMGZ0LiUyMFNpYSZ0eXBlPWxyY2RkldCkUb4gqLLarJTey5JNJCWNbyGG7G%2FEUyKfIaQ4lZ8%3D'
```

**Review by PedroHLC — APPROVED** [review]
Well, worked nicely here.
But B.Y.O.B got all censored :rofl: 

Maybe, but in a future PR, is better to prepare the cookies and thus simulating a user pressing the download button.

**Comment by dasj19** [comment]
Thanks for the advice. I think it is safe to "visit" a website with a scraper, maybe we should spoof the user agent to look like a common browser. But I'll look into using the "Dowload" button when time allows.

How about the print statements? I promised to remove them, or should we do that before release

**Comment by PedroHLC** [comment]
> How about the print statements? I promised to remove them, or should we do that before release

Yeah, sorry, I forgot about those :sweat:

**Pull request merged**